src/main/java/commands/WorkoutCommand.java

```diff
@@ -95,6 +94,11 @@ public void execute() {
                 getUI().printDeleteWorkoutMessage(deletedWorkout);
                 getFileManager().rewriteAllWorkoutsToFile(getWorkoutList());
                 break;
+            case UPDATE_ACTION_KEYWORD:
+                Workout updateWorkout = getWorkoutList().updateWorkout(getUserArguments());
```

Comment:
Just a minor naming suggestion: perhaps `updateWorkout` can be renamed to `updatedWorkout`? To better indicate that this variable will store the `Workout` object that has been updated. 🙂

Reply:
Thank you for your suggestion.